### ISSUE TITLE
[Bug] Load SVG but radialGradient not work

### ISSUE THREAD

**Issue by tctdrb** [apache/echarts#18407]
### Version

5.4.1

### Steps to Reproduce

I am using echarts to load SVG , but radialGradient not work.

![image](https://user-images.githubusercontent.com/29109418/226829173-23e608ad-6bb4-48b5-a912-d1123e0a0162.png)

<!DOCTYPE html>
<html lang="zh-CN" style="height: 100%">

<head>
    <meta charset="utf-8">
</head>

<body style="height: 100%; margin: 0">

    <svg id="svg2" width="400px" height="400px">
        <defs>
            
            <radialGradient id="rg2" cx="30%" cy="30%" r="50%" fx="50%" fy="50%">
                <stop offset="0" style="stop-color: chocolate;"></stop>
                <stop offset="0.3" style="stop-color: cadetblue;"></stop>
                <stop offset="0.5" style="stop-color: chartreuse;"></stop>
                <stop offset="0.7" style="stop-color: #123456"></stop>
                <stop offset="1" style="stop-color: lightpink;"></stop>
            </radialGradient>
        </defs>

        
        <circle cx="200" cy="200" r="100" style="fill: url(#rg2);"></circle>
    </svg>

    <div id="container" style="height: 100%"></div>

    <script type="text/javascript" src="https://fastly.jsdelivr.net/npm/jquery"></script>
    <script type="text/javascript" src="https://fastly.jsdelivr.net/npm/echarts@5.4.1/dist/echarts.min.js"></script>

    <script type="text/javascript">
        var dom = document.getElementById('container');
        var myChart = echarts.init(dom, null, {
            renderer: 'canvas',
            useDirtyRect: false
        });
        var app = {};
        var option;

        $.get('./a.svg', function (svg) {
            echarts.registerMap('flight-seats', { svg: svg });
            option = {
                tooltip: {},
                geo: {
                    map: 'flight-seats',
                    roam: true,
                    selectedMode: 'multiple',
                    layoutCenter: ['50%', '50%'],
                    layoutSize: '100%',
                    tooltip: {
                        show: true
                    },
                    itemStyle: {
                        color: '#fff'
                    },
                    emphasis: {
                        itemStyle: {
                            color: undefined,
                            borderColor: 'green',
                            borderWidth: 2
                        },
                        label: {
                            show: false
                        }
                    },
                    select: {
                        itemStyle: {
                            color: 'green'
                        },
                        label: {
                            show: false,
                            textBorderColor: '#fff',
                            textBorderWidth: 2
                        }
                    }
                }
            };

            myChart.setOption(option);
        });

        if (option && typeof option === 'object') {
            myChart.setOption(option);
        }

        window.addEventListener('resize', myChart.resize);
    </script>
</body>

</html>

### Current Behavior

I am using echarts to load SVG , but radialGradient not work.

### Expected Behavior

 Make radialGradient working.

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by helgasoft** [comment]
agree, [Demo Code](https://echarts.apache.org/examples/en/editor.html?c=line-simple&code=MYewdgzgLgBBEF4YwOQB4IDcDmMCWAJggERbYBMxMA7oVABYkAsADCwA4AeV9ApntnpRmbLsQB8MNAV4AzCJKkAnAIYE8KgDYBxVet5hYhEkopVgnEgGYWAUnMBPa3apKSAVhcxZl4p_veTn4uilLQIOwwILLyvMLELFTQDpq8JOHsALSgmiBKAFwwwPQgOSpQvADcEmgA9BmhGFARUTEQcSQsAHRWSVApaaTNWTl5hcBqcQBGmgCuVTX1w6HIq2tNLdGx8d3ufQPpw9kguQVF9CpKUEq8s-3V4nUNay-rG5Fb7TtdAOz7qYcIsdToUAMQARnIViY7gAbItnsh3q1tiRwf9BhlgWMYJoBEJ2HgwABrB5PZZI2p6DQ6akGKArOoyeSM4B4JTAVJFXzkNiOEi8xIwNzEcF8uD9AHEWR4TSaQqzJSaAAUoNM5AAlGTamyOalHvUcOIUJUAFC8YqXKAQLo3bB4aC8JQAWRU7GVKHa5QgKAANDAAN5wHCFeAwAC-WtNESgeHAMCQAdNyHaSjwvAghSTa367F4hRQAFs3X7k6ti-wC17raW1koQCpC4VrvNfWXkJoVA4QLMoABhelOwoAbRQ_j9qHHAF028hw6bw5UgA)

**Comment by plainheart** [comment]
Currently, parsing from SVG is still limited. The case you mentioned is not supported yet. See also https://github.com/ecomfe/zrender/blob/master/src/tool/parseSVG.ts#L605-L641
I believe it would be awesome if we enhanced this feature in the future.

**Comment by ssjtbcf** [comment]
@plainheart  linearGradient is not work too, please enhance it in the future!!

**Comment by ssjtbcf** [comment]
> Currently, parsing from SVG is still limited. The case you mentioned is not supported yet. See also https://github.com/ecomfe/zrender/blob/master/src/tool/parseSVG.ts#L556-L596 I believe it would be awesome if we enhanced this feature in the future.

 linearGradient is not work too, Do you have a plan for when to support?

**Comment by wangchaoHeart** [comment]
linearGradient is not work too, Do you have a plan for when to support?

**Comment by helgasoft** [comment]
@ssjtbcf, @wangchaoHeart - linearGradient seems to work in v.5.6.0 - [Demo](https://echarts.apache.org/examples/en/editor.html?c=line-simple&code=M4ZgvABABgPMBuBzCAPAtgGwHbDAIgAsAXIgBwC4B6SgdzoDoaR6B7AJ0UoCYAGPyhIjwR4ASwCmNAEIsU-HhAVcAjBBV4AfACgYAE3EAzYNogQYGUVnEBDNgHE213RKxEIo3fkRth3py6IAVSxRIlw8AFdgcTYAZVJrAGNxAHksQOjNLVNTOCIWUggWAyNxInlhYHzSAFpElgx2fAAjDAjxPEoTHLyCopLo8rxlSuq6hqa8C0RiUksAa07umEoLK1sHf3FXbuyzBJIYrHdPPFIiYTFJGTk8HgAaB-UnnmEaDyICfGeAUmECcSiGZDX5ZHL7BoAT0QLGOpBYljC8keanuAFZFPdnhA0fcABwQZ5YhR49GEh4KXFcYRdPYrA5EI7aFb6IzMtjiRJuW4jCDvXSffAAFiF_0BwPw1IgVUhGHE-AMogwGHIETYGAAFABibwASiWOg5XIgkPweLeHy-w1FEABQOIwtGsvleEVytV6u1511AG5pUQ2Cx5uJyBzdAaVoINFAtJyCLYwvQOYhRFUYgBZaykDUAcjQghz9wgAG9pUhyNKQBAAL6-rQFIiiWEQSDFvbRNgSYAVtvgoiQ0ghiB5rOFvamNBZit5gv3ccQQPWNAVgPtOemataas-oA)

**Comment by wangchaoHeart** [comment]
Thank you very much for your answer. I added a “stop-opacity” attribute to your [example](https://echarts.apache.org/examples/en/editor.html?c=line-simple&code=M4ZgvABABgPMBuBzCAPAtgGwHbDAIgAsAXIgBwC4B6SgdzoDoaR6B7AJ0UoCYAGPyhIjwR4ASwCmNAEIsU-HhAVcAjBBV4AfACgYAE3EAzYNogQYGUVnEBDNgHE213RKxEIo3fkRth3py6IAVSxRIlw8AFdgcTYAZVJrAGNxAHksQOjNLVNTOCIWUggWAyNxInlhYHzSAFpElgx2fAAjDAjxYUoTHLyCopLo8rxlSuq6hqa8NnFdTu6YSgsrWwd_cVdu7LMEkhisd088UiJhMUkZOTweABob5TueYRoPIgJ8e4BSYQJxUURid48L7dXKkBoAT0QLH2YMsYXktzU1wArIprvcIMjrgAOCD3dEKbEovE3BRYrhzLYLHZEPbaBb6Iz06aJNyXEYQZ66V74AAsvO-v3-QwpECq4Iw4nwBlEGAw5AibAwAAoAMTeACUeC6Wh0AiQGiguuoECWNnsjmc6yIgCg5QC0clU-oAQt0dtQKSVC4MAsHKAfFdABKmWmAvIgYFgglQmBw-GIZCotAYTFYHG4fB4-qEIgk0lk8kUalU6npjOMVLNK0tAQOXh8ED8VtcwVC4SiMXiSVS6UyILMrv6pSGjzFY3qjTYLTaHVMrpq7sSnvk9GR2u7vUKxX771GBXGo_w01mQ-3s_nV2XVMWlnNq3rRHplGLzPErNQm85LzeeH5gr-ALwovFkrSrK8qKiq6psFqOoLIIhq6nBT4ELYYT0NMiCiFUMQALLWKQyoAORoIIeHXBAADeYpIOQYq8loAC-GoANxaAURCiNCIZkVs0RsBIwBUaRWymEQ4KkOIVEEThxGCRAaA4eJhFIFJOQQGwLDWGgVFEGw7TXFstF0QxQA), but it did not take effect

**Comment by helgasoft** [comment]
@wangchaoHeart, take a look at the _parseSVG.ts_ link by @plainheart. SVG parsing is still very basic.
